src/storage/StorageServiceHandler.cpp

```diff
@@ -61,14 +62,17 @@ StorageServiceHandler::future_getProps(const cpp2::VertexPropRequest& req) {
     auto* processor = QueryVertexPropsProcessor::instance(kvstore_,
                                                           schemaMan_,
                                                           &vertexPropsQpsStat_,
-                                                          getThreadManager(),
+                                                          readerPool_.get(),
                                                           &vertexCache_);
     RETURN_FUTURE(processor);
 }
 
 folly::Future<cpp2::EdgePropResponse>
 StorageServiceHandler::future_getEdgeProps(const cpp2::EdgePropRequest& req) {
-    auto* processor = QueryEdgePropsProcessor::instance(kvstore_, schemaMan_, &edgePropsQpsStat_);
+    auto* processor = QueryEdgePropsProcessor::instance(kvstore_,
+                                                        schemaMan_,
+                                                        &edgePropsQpsStat_,
+                                                        readerPool_.get());
     RETURN_FUTURE(processor);
```

Comment:
Should the following  `future_getEdgeKeys `and `future_getUUID ` functions also be processed?

Reply:
Good point.